docs/00-Getting-started/02-Overview.md

```diff
@@ -0,0 +1,128 @@
+# Overview
+
+AgnosUI is a versatile widget library designed for various frameworks, providing a consistent user interface across platforms. Its core (also referred to as headless), written in pure TypeScript, serves as a foundation that can be utilized by any framework to generate a user interface.
+
+You can integrate AgnosUI in two primary ways:
+
+- Use the ready-made widgets provided for your specific framework.
+
+This straightforward approach allows you to quickly incorporate widgets into your project. Refer to the component documentation for usage details. Currently, AgnosUI supports Angular, React, and Svelte.
+
+- use the core functionality to create your own customized widget.
```

Comment:
you mean to use headless or the core APIs ?

Reply:
Thanks, I will use headless as it's more appropriate when talking about creating a widget on his own